src/routes/tabStack.tsx

```diff
@@ -215,13 +213,10 @@ function TabStack() {
           return (
             <CyDAnimatedView
               // TO REDO : TABBAR ANIMATION
-              entering={SlideInDown}
-              exiting={SlideOutDown}
-              layout={Layout.easing(Easing.ease).delay(50)}
-              className={clsx('rounded-t-[24px] shadow', {
-                'mb-[-90px]': !showTabBar,
-                'absolute bottom-[-350px]': keyboardHeight,
-                'bg-white': !isIOS(),
+              layout={Layout.springify()}
```

Comment:
Change it to easing

Reply:
![Simulator Screen Recording - iPhone 14 - 2023-09-05 at 11 33 18](https://github.com/CypherD-IO/mobile-app/assets/132334547/55d2fa40-ff1e-4612-91f3-19831eaa6d35)